test/Form_test.js

```diff
@@ -229,6 +229,27 @@ describe("Form", () => {
       expect(node.querySelectorAll("input[type=text]")).to.have.length.of(1);
     });
 
+    it("should handle references to deep schema definitions", () => {
+      const schema = {
+        definitions: {
+          testdef: {
+            type: "object",
+            properties: {
+              bar: { type: "string" },
+            },
+          },
+        },
+        type: "object",
+        properties: {
+          foo: { $ref: "#/definitions/testdef/properties/bar" },
```

Comment:
Actually I couldn't find any reference in the spec about this capability. The only part of it using nested definitions is [this one](http://json-schema.org/latest/json-schema-core.html#rfc.section.8.2) but I'm not sure about accessing `properties` like your implemention does would actually comply.

Do you by chance have a link describing how implementations should access nested definitions?